game/src/main/scala/io/aigar/game/Cell.scala

```diff
@@ -35,7 +42,7 @@ class Cell(val id: Int, player: Player, startPosition: Vector2 = new Vector2(0f,
    * second.
    */
   def maxSpeed: Float = {
-    50f //TODO depend on mass?
+    max(50f - mass*0.002, Cell.MinSpeed).toFloat
```

Comment:
Could we have those as constants?
Suggestions:
`MaxSpeedLimit = 50f`
`SpeedLimitReductionPerMassUnit = 0.002f`

Also, by changing `0.002` to `0.002f` I don't think we need `toFloat` anymore.